pkg/cincinnati/cincinnati_test.go

```diff
@@ -745,7 +745,7 @@ func TestGetUpdates(t *testing.T) {
 			ts := httptest.NewServer(http.HandlerFunc(handler))
 			defer ts.Close()
 
-			c := NewClient(clientID, nil, "")
+			c := NewClient(clientID, nil, "", standard.NewConditionRegistry(nil))
```

Comment:
nit: I'd prefer if the registry had a `WithClient` variant or a fluent method but I guess it's just a cosmetic thing